pkg/app/app.go

```diff
@@ -158,7 +163,8 @@ func (app *App) Run() error {
 		os.Exit(0)
 	}
 
-	return app.Gui.RunWithSubprocesses()
+	err := app.Gui.RunWithSubprocesses()
```

Comment:
Why not just `return app.Gui.RunWithSubprocesses()`?

Reply:
I agree, I was just following this approach, happy to change it back: https://github.com/jesseduffield/lazydocker/pull/14/files#diff-637a60a871f8b69622bdfd3d3b2fcad1

Reply:
I think that is there for later use or jesseduffield forgot to change it back

Reply:
Do you think we should leave it as it is so the two projects are in sync, given that they are pretty similar?

Reply:
Hi @mjarkk are you happy to merge this?